### PR TITLE
Remove the static man page in favor of the automatically generated one

### PR DESCRIPTION
**Pull request by wilzbach**
Follow-up to https://github.com/dlang/dmd/pull/7637
Not sure whether the nightlies depend on the exact path.

CC @MartinNowak

### CODE REVIEW

**Review by JinShil — APPROVED** [review]

**Comment by wilzbach** [comment]
(I'm assigning this to @MartinNowak, s.t. it's clear that he should approve this PR before being merged.)

**Comment by MartinNowak** [comment]
Well, the build seems to work already.
http://nightlies.dlang.org/dmd-master-2018-01-25/build.html
```
../generated/linux/release/32/gen_man > ../generated/docs/man/man1/dmd.1
```
Copying the result is still missing though https://github.com/dlang/installer/blob/1d0c5445ba4f1ff948fe48bc1ab27c3d90f1c2c8/create_dmd_release/create_dmd_release.d#L601.

**Comment by wilzbach** [comment]
> Well, the build seems to work already.

Cool! But I assume this is just there because the man page generation happens as part of the `install` target.

> Copying the result is still missing though 

Fair enough: https://github.com/dlang/installer/pull/295

**Pull request merged**
